mixer/tools/mixgen/cmd/adapter.go

```diff
@@ -0,0 +1,114 @@
+// Copyright 2018 Istio Authors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package cmd
+
+import (
+	"bytes"
+	"encoding/base64"
+	"io/ioutil"
+	"path/filepath"
+	"strings"
+	gotemplate "text/template"
+
+	"github.com/spf13/cobra"
+
+	"istio.io/istio/mixer/cmd/shared"
+)
+
+const istioSystemNamespace = "istio-system"
```

Comment:
instead of adding another hardcoded var, should we try to reuse the exported var from runtime for this: https://github.com/istio/istio/blob/66434029ad81f1d6e049c413c5150908d147d9d1/mixer/pkg/runtime/runtime.go#L40 ?